include/drivers/pcie/pcie.h

```diff
@@ -39,6 +39,72 @@ typedef uint32_t pcie_id_t;
  * These functions are arch-, board-, or SoC-specific.
  */
 
+/**
+ * @brief Read a 8-bit word from an endpoint's configuration space.
+ *
+ * This function is exported by the arch/SoC/board code.
+ *
+ * @param bdf PCI(e) endpoint
+ * @param reg byte offset from the beginning of the configuration space
+ * @return the word read (0xFFU if nonexistent endpoint or word)
+ */
+extern uint8_t pcie_conf_read_8(pcie_bdf_t bdf, unsigned int reg);
```

Comment:
this is byte offset, probably better name as `addr` opposite to 32bit register number as we have for other functions